README.md

```diff
@@ -10,8 +10,3 @@ Use `xsd-isogen in.xsd Generator > out.hs` for queries.
 Use `xsd-isogen in.xsd Parser > out.hs` for responses.
 
 Use `xsd-isogen in.xsd Both > out.hs` for common definitions.
-
-Keep in mind that name clashes are possible and should be disambiguated
```

Comment:
Should be substituted with a description of disambiguation renaming process.

Reply:
Done in #2